internal/pkg/cli/job_init.go

```diff
@@ -0,0 +1,297 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"encoding"
+	"fmt"
+	"strings"
+
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/aws/copilot-cli/internal/pkg/term/color"
+	"github.com/aws/copilot-cli/internal/pkg/term/log"
+	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
+	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	fmtJobInitNameHelpPrompt = `The name will uniquely identify this job within your app %s.
+Deployed resources (such as your job, logs) will contain this job's name and be tagged with it.`
+
+	jobInitDockerfileHelpPrompt = "Dockerfile to use for building your job's container image."
+)
+
+// const (
+// 	fmtAddJobToAppStart    = "Creating ECR repositories for job %s."
+// 	fmtAddJobToAppFailed   = "Failed to create ECR repositories for job %s.\n"
+// 	fmtAddJobToAppComplete = "Created ECR repositories for job %s.\n"
+// )
+
+type initJobVars struct {
```

Comment:
Do we want to include the `Rate`, `Retries`, and `Timeout` fields mentioned in the Example section of the command?